docs/start/setup-channel.mdx

```diff
@@ -165,7 +165,72 @@ print(result)
     from qiskit_ibm_runtime import QiskitRuntimeService
 
     service = QiskitRuntimeService()
-    ```
+  ```
+
+## Set up account using REST APIs
+
+Alternatively, you can also access IBM Quantum Systems using REST APIs, enabling you to work with quantum systems from any programming language or framework. 
+
+1. Retrieve your IBM Quantum token from the [IBM Quantum account page](https://quantum.ibm.com/account) and save it to a file named `token`.
+
+2. Obtain a temporary Access token by supplying the API token obtained above. This is especially useful if you would like control over tokens, such as token invalidation. Alternatively, you can work directly with your IBM Quantum Platform API token.
+
+```python
```

Comment:
<img width="879" alt="Screenshot 2024-06-14 at 12 08 15 PM" src="https://github.com/Qiskit/documentation/assets/23662430/91ddac37-4cda-4491-a7a5-b42ff2dba4f4">

something looks a bit off with the spacing of the code blocks, they should be indented so they look like they are within the numbered text, not overflowing like this. See how the spacing is done for the other sections already in this page